app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -442,30 +436,40 @@ class _ReplyContentWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    if (msg is TextMessage) {
-      return messageWidget!;
-    } else if (msg is ImageMessage) {
+    if (msg is ImageMessage) {
+      var imageMsg = msg as ImageMessage;
       return Padding(
         padding: const EdgeInsets.all(8.0),
-        child: ConstrainedBox(
-          constraints: const BoxConstraints(maxHeight: 100, maxWidth: 125),
-          child: ClipRRect(
-            borderRadius: BorderRadius.circular(6.33),
-            child: Image.memory(
-              base64Decode(msg?.metadata?['base64']),
-              fit: BoxFit.fill,
-              cacheWidth: 125,
-            ),
-          ),
+        child: ImageMessageBuilder(
+          message: imageMsg,
+          messageWidth: imageMsg.size.toInt(),
+          isReplyContent: true,
         ),
       );
-    } else if (msg is FileMessage) {
-      return messageWidget!;
-    } else if (msg is CustomMessage) {
-      return messageWidget!;
-    } else {
-      return const SizedBox.shrink();
+    } else if (msg is TextMessage) {
+      var textMsg = msg as TextMessage;
+      bool enlargeEmoji = false;
+      if (textMsg.metadata!.containsKey('enlargeEmoji')) {
+        enlargeEmoji = textMsg.metadata!['enlargeEmoji'];
+      }
+      return Container(
+        alignment: Alignment.center,
+        constraints: const BoxConstraints(maxHeight: 120),
+        padding: const EdgeInsets.all(8),
+        child: enlargeEmoji
+            ? Text(textMsg.text, maxLines: 3, overflow: TextOverflow.ellipsis)
+            : Html(
+                // ignore: prefer_single_quotes, unnecessary_string_interpolations
+                data: """${textMsg.text}""",
```

Comment:
yeah, but why though?  why not
```suggestion
                data: textMsg.text,
```

Reply:
I think that's how supposed to be according to package examples. But its working fine without adding double quotes. I will change to it.